From ebab059dae29b0983cdde99d6f11bffdb1655c24 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Thu, 11 Sep 2025 18:10:59 +0200
Subject: [PATCH 093/N] configure: Work around broken pc file

Invalid "Libs:" value

Also it contains "-L." which is only used at build time by
makesetup and shouldn't be exposed.

https://github.com/python/cpython/issues/138800
---
 configure.ac | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index 9bab391..fa8eb8c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -6799,6 +6799,11 @@ LIBPYTHON=''
 if test "$PY_ENABLE_SHARED" = "1" && ( test -n "$ANDROID_API_LEVEL" || test "$MACHDEP" = "cygwin" || test "$MACHDEP" = "win32"); then
   MODULE_DEPS_SHARED="$MODULE_DEPS_SHARED \$(LDLIBRARY)"
   LIBPYTHON="\$(BLDLIBRARY)"
+  LIBPYTHON="$BLDLIBRARY"
+  LIBPYTHON=$(echo "$LIBPYTHON" | sed "s/\$(LDVERSION)/$LDVERSION/g")
+  LIBPYTHON=$(echo "$LIBPYTHON" | sed "s/\$(VERSION)/$VERSION/g")
+  LIBPYTHON=$(echo "$LIBPYTHON" | sed "s/\$(ABIFLAGS)/$ABIFLAGS/g")
+  LIBPYTHON=$(echo "$LIBPYTHON" | sed "s/-L\. //g")
 fi
 
 # On iOS the shared libraries must be linked with the Python framework
